Source/Common/CMakeLists.txt

```diff
@@ -22,3 +22,7 @@ endif()
 add_library(${NAME} STATIC ${SRCS})
 target_link_libraries(${NAME} FEXCore_Base cpp-optparse tiny-json FEXHeaderUtils range-v3::range-v3)
 target_include_directories(${NAME} PRIVATE ${CMAKE_BINARY_DIR}/generated)
+
+set_target_properties(${NAME} PROPERTIES C_VISIBILITY_PRESET hidden)
```

Comment:
Should these be coalesced like:

```cmake
set_target_properties(${NAME} PROPERTIES
  C_VISIBILITY_PRESET hidden
  CXX_VISIBILITY_PRESET hidden
  VISIBILITY_INLINES_HIDDEN TRUE
)
```

or something of the sort, so any properties added in the future just requires adding only the key and value in a convenient way